src/legacy/core_plugins/kibana/public/context/components/action_bar/action_bar.tsx

```diff
@@ -0,0 +1,146 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import {
+  EuiButtonEmpty,
+  EuiFieldNumber,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFormRow,
+  EuiSpacer,
+} from '@elastic/eui';
+import { ActionBarWarning } from './action_bar_warning';
+
+export type ActionBarType = 'successor' | 'predecessor';
+
+export interface ActionBarProps {
```

Comment:
really nice documentation 👍

Reply:
merci

> Sometimes a strange title pops up when typing in the size field:
> ![strange_title](https://user-images.githubusercontent.com/1508364/64428861-10bd2500-d0b5-11e9-8623-05fe3d73c95b.gif)
> 
> Why is 209 not valid, but 208 and 213?

this is pretty odd, just replaced a plain input box with EUI thx, will investigate this